### PR TITLE
[docs] Update README example

### PR DESCRIPTION
**Pull request by SHtress**
Fix for #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,44 +12,82 @@
...
-- components install [--idtf \<system_idtf\>] - installing component by it's system identifier.  
+### Commands
 
-## Repository and components
+- `components init` - downloading specifications from repositories. `kb/specifications.scs` contains example of how to describe repository.
+- `components search  [--author \<author\>][--class \<class\>][--explanation \<"explanation"\>]` - searching component specification in knowledge base. You can search components by author, class or explanation substring.
+- `components install [--idtf \<system_idtf\>]` - installing component by it's system identifier.  
 
-File specification.scs contains description of two sections: `components` and `repositories`.
+## Repository and components
```
Comment (MksmOrlov) [inline README.md:28]: There is no `Repository`, we name it as a `Storage` now

File: README.md [hunk 2]
```diff
@@ -12,44 +12,82 @@
...
-`Repositories` have links for source (GitHub, google drive etc.) with specification file, `components` have links to source with specification of component.
+File specification.scs contains description of two sections: **components** and **repositories**.
 
+**Repositories** have links for source (GitHub, google drive etc.) with specification file, **components** have links to source with specification of component.
 
 ### Repository specification
 
 Example of repository (`specifications.scs`)
 
 ```scs
```
Comment (MksmOrlov) [inline README.md:38]: Add SCg example like that
image

File: README.md [hunk 3]
```diff
@@ -12,44 +12,82 @@
...
+      -> rrel_address:
+          [https://github.com/MksmOrlov/components-repo-example];;
+      *);;
 ```
 
 ### Component specification
+
 Example of components specification (`specification.scs`)
 
 ```scs
```
Comment (MksmOrlov) [inline README.md:93]: add SCg example

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SHtress) [reply README.md:28]: Maybe, but until it changed everywhere it stays as Repository. Not for this PR.

Comment (SHtress) [reply README.md:28]: Unnecessarily. Can be added to documentation but not to README. If you want to change all examples from `scs` to `scg` create issue.

File: README.md (hunk 2 above)
Comment (SHtress) [reply README.md:38]: Unnecessarily. Can be added to documentation but not to README. If you want to change all examples from `scs` to `scg` create issue.

File: README.md (hunk 3 above)
Comment (SHtress) [reply README.md:93]: Also i try to create one PR for one feature or fix.

**Review by smousy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MksmOrlov) [reply README.md:28]: https://github.com/ostis-ai/sc-component-manager/issues/38

File: README.md (hunk 2 above)
Comment (MksmOrlov) [reply README.md:38]: ok but SCn documentation MUST HAVE SCg examples

**Review by MksmOrlov — APPROVED** [review]

**Pull request merged**